### PR TITLE
Textbasedgame final class submission

### PR DESCRIPTION
**Pull request by dmcphers**

### CODE REVIEW

File: textbasedgame/Program.cs [hunk 1]
```diff
@@ -0,0 +1,187 @@
+﻿using System;
+
+namespace textbasedgame
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // assign variables
+            string ch1;
```
Comment (daniefer) [inline textbasedgame/Program.cs:10]: In general, I would say it is best to declare and use variables closer to where they are relevant. We are not doing JavaScript in this class, so no need to do this.

**Review by daniefer — APPROVED** [review]
Great job!

**Comment by daniefer** [comment]
Be careful where you branch from. I am seeing hello world assignment files in this pull request. Make sure you always change to master before making a new branch.

**Comment by dmcphers** [comment]
Thanks. Yeah that's what I was referring to when I wrote you on rocket chat
that time. Not sure how that happened. think I know the right way to do it,
so hopefully it won't happen again :)

**Pull request merged**